
public class ThrowDemo {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
		int x =2,y = 0;
		try {
			
			if(y == 0);
			{
				throw new ArithmeticException();
				
			}}
			catch(ArithmeticException e)
			{
				System.out.println("Exception Occured: Division by zero");
			}
		}

}
